marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-match-merge-writer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "role-name": "hub-central-match-merge-writer",
+  "description": "Permits reading/reviewing matching and merging steps in Hub Central",
+  "role": [
+    "hub-central-match-merge-reader",
+    "data-hub-common-writer",
```

Comment:
"data-hub-match-merge-writer" already has "data-hub-common-writer". So, this can be removed.

Reply:
Removed this